kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java

```diff
@@ -23,6 +23,8 @@
   DOCUMENTS(true, "documents"),
   TAGGED(true, "tagged"),
 
+  // Edges from an anchor.
+  IMPUTES("imputes"),
```

Comment:
It needs to use the other constructor, that's what signifies it as an anchor edge: `IMPUTES(true, "imputes")`.

Reply:
done